rest/networkProtocols/TheThingsNetwork.js

```diff
@@ -17,25 +17,37 @@ module.exports = {
       oauthUrl: 'https://account.thethingsnetwork.org/users/authorize',
       protocolHandlerNetworkFields: [
         {
-          fieldName: 'clientId',
-          fieldDesc: 'Client Id',
-          fieldHelp: 'The client id chosen when registering LPWan',
-          fieldType: 'string',
-          displayWithQueryParameter: ''
+          name: 'clientId',
+          description: 'Client Id',
+          help: 'The client id chosen when registering LPWan',
```

Comment:
can be omitted for now, at some point we can populate w info RE how to generate client id